source/crud/write-operations/arrayFilters.txt

```diff
@@ -0,0 +1,112 @@
+===========================
+Update Arrays in a Document
+===========================
+
+.. default-domain:: mongodb
+
+To modify arrays nested within a document, use the
```

Comment:
nested -> embedded

This is a general MongoDB style guide across all of our messaging